modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -65,24 +65,49 @@ export interface AcceptShareOptions {
 }
 
 export interface AddWalletOptions {
-  type?: string;
+  coinSpecific?: any;
+  enterprise?: string;
+  isCold?: IsCold;
+  isCustodial?: IsCustodial;
   keys?: string[];
+  keySignatures?: KeySignatures;
+  label: string;
+  multisigType?: 'onchain' | 'tss' | 'blsdkg';
+  address?: string;
   m?: number;
   n?: number;
   tags?: string[];
+  type?: string;
+  walletVersion?: number;
+  eip1559?: Eip1559;
   clientFlags?: string[];
```

Comment:
I've noticed that all parameters from `clientFlags` and down aren't actually specified on the API documentation.

Reply:
That could be because the API documentation is out of date, or the implementation was removed. Best to lookup the route in wallet-platform & verify the params if we're unsure here.

Reply:
Yup, they're undocumented deeply nested params.